loadtest/control/actions.go

```diff
@@ -190,73 +208,84 @@ func AddReaction(u user.User) UserActionResponse {
 	// get posts from UserStore that have been created in the last minute
 	posts, err := u.Store().PostsSince(time.Now().Add(-1*time.Minute).Unix() * 1000)
 	if err != nil {
-		return UserActionResponse{Err: err}
+		return UserActionResponse{Err: NewUserError(err)}
 	}
 	if len(posts) == 0 {
 		return UserActionResponse{Info: "no posts to add reaction to"}
 	}
 
+	post := posts[rand.Intn(len(posts))]
+
 	err = u.SaveReaction(&model.Reaction{
 		UserId:    u.Store().Id(),
-		PostId:    posts[0].Id,
+		PostId:    post.Id,
 		EmojiName: "grinning",
 	})
 
 	if err != nil {
-		return UserActionResponse{Err: err}
+		return UserActionResponse{Err: NewUserError(err)}
 	}
 
-	return UserActionResponse{Info: "added reaction"}
+	return UserActionResponse{Info: fmt.Sprintf("added reaction to post %s", post.Id)}
 }
 
 func RemoveReaction(u user.User) UserActionResponse {
```

Comment:
Should we rename this to `RemoveReactions` then ? Since it is actually removing multiple reactions.

Reply:
I believe it's either removing one or none if it can't find any. We immediately return after removing.

Reply:
Oh my bad. Yes that's right.